src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java

```diff
@@ -149,6 +149,27 @@ public void execute(Boolean succeeded)
         }
       );
    }
+   
+   public void withRPlumber(String userAction, final Command command)
+   {
+      withDependencies(
+        "Plumber",
+         userAction,
+         new Dependency[] {
+            Dependency.cranPackage("plumber", "0.4.5")
```

Comment:
Do we expect that 0.4.5 will be on CRAN soon? If not let's embed this package so daily users can use the plumber integration (ping me for details, should be very straightforward!)